assets/src/setup/setup-wizard.js

```diff
@@ -49,17 +53,37 @@ function Page( { children, exitLink } ) {
 	return children;
 }
 
+/**
+ * Side effect wrapper for page component.
+ *
+ * @param {Object} props Component props.
+ * @param {?any} props.children Component children.
+ */
+function PageComponentSideEffects( { children } ) {
+	useEffect( () => {
+		document.body.scrollTop = 0;
+		document.documentElement.scrollTop = 0;
+	}, [] );
+
+	return children;
+}
+
 /**
  * Setup wizard root component.
  *
  * @param {Object} props Component props.
  * @param {Array} props.pages List of page configuration objects.
```

Comment:
Docblock is outdated and should include `exitLink` instead of `pages`.
```suggestion
 * @param {Array} props.exitLink Exit link.
```